### PR TITLE
SDK-305: Integrate structured_postal_address attribute feature

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: lib/yoti/activity_details.rb [hunk 1]
```diff
@@ -42,5 +42,12 @@ def initialize(receipt, decrypted_profile = nil)
       @user_id = receipt['remember_me_id']
       @outcome = receipt['sharing_outcome']
     end
+
+    public
+        # @return a JSON [Hash] of the address
```
Comment (echarrod) [inline lib/yoti/activity_details.rb]: Could we change this so it's like the other attributes on this class with the type first, e.g. 
` # @return [Hash] a JSON [Hash] of the address`

Comment (sidibos) [reply lib/yoti/activity_details.rb]: That looks like a redundant to me.

**Comment by sidibos** [comment]
Regenerating Attribute.proto won't help in this case. I am adding a support for a type that's coming from connect so it can be converted accordingly.
Also, all protobuf instances will be upgraded in the next task, when implementing anchor processing. It's not needed for this one.

File: lib/yoti/activity_details.rb (hunk 1 above)
Comment (echarrod) [reply lib/yoti/activity_details.rb]: Sorry, I meant `# @return [Hash] a JSON of the address`

Comment (sidibos) [reply lib/yoti/activity_details.rb]: That makes sense.

File: lib/yoti/protobuf/v1/attribute_public_api/attribute.pb.rb [hunk 2]
```diff
@@ -13,6 +13,7 @@ class ContentType < ::Protobuf::Enum
           define :JPEG, 2
           define :DATE, 3
           define :PNG, 4
+          define :BYTES, 5
```
Comment (qzagarese) [inline lib/yoti/protobuf/v1/attribute_public_api/attribute.pb.rb]: is this auto generated or you wrote it yourself? If it's the latter, why bytes and not json?

File: lib/yoti/protobuf/v1/protobuf.rb [hunk 3]
```diff
@@ -10,6 +11,7 @@ class << self
       CT_JPEG = 2 # standard .jpeg image.
       CT_DATE = 3 # string in RFC3339 format (YYYY-MM-DD)
       CT_PNG = 4 # standard .png image
+      CT_BYTES = 5 # json_string
```
Comment (qzagarese) [inline lib/yoti/protobuf/v1/protobuf.rb]: same here

File: lib/yoti/protobuf/v1/protobuf.rb [hunk 4]
```diff
@@ -31,6 +33,8 @@ def value_based_on_content_type(value, content_type = nil)
           value.encode('utf-8')
         when CT_DATE
           value.encode('utf-8')
+        when CT_BYTES
```
Comment (qzagarese) [inline lib/yoti/protobuf/v1/protobuf.rb]: same here

File: lib/yoti/protobuf/v1/attribute_public_api/attribute.pb.rb (hunk 2 above)
Comment (sidibos) [reply lib/yoti/protobuf/v1/attribute_public_api/attribute.pb.rb]: Yes I wrote it myself, I put bytes in reference to Node but will change it to JSON.

**Pull request merged**